quinn-proto/src/crypto/rustls.rs

```diff
@@ -326,7 +323,12 @@ fn to_vec(params: &TransportParameters) -> Vec<u8> {
 }
 
 pub(crate) fn initial_keys(version: Version, dst_cid: &ConnectionId, side: Side) -> Keys {
-    let keys = rustls::quic::Keys::initial(version, dst_cid, side.is_client());
+    let side = if side.is_client() {
+        rustls::Side::Client
+    } else {
+        rustls::Side::Server
+    };
```

Comment:
This was a quick fix. I think probably it would be nicer to add a `From<Side> for rustls::Side` impl to convert between the Quinn `Side` and the Rustls `Side` :thinking:

Reply:
Tidied up in [bf15998](https://github.com/quinn-rs/quinn/pull/1515/commits/bf159983708465eaa80ba7d5ca2745bf33ad3d50).